gourmet/convert.py

```diff
@@ -11,29 +12,61 @@
 
 USE_FRACTIONS = FRACTIONS_NORMAL
 
-class PossiblyCaseInsensitiveDictionary (dict):
+class PossiblyCaseInsensitiveDictionary(collections.abc.MutableMapping):
+    # Maps all capitalisation variants of a key to a single value
+    # Invariant 1: two keys that have a variant in common, map to the same value (__mapping)
+    # Invariant 2: a key that is added can be recalled in its original (__orig)
+    # Invariant 3: keys(__orig) == keys(__mapping)
```

Comment:
Maybe we could have a PEP 257 docstring? ;-)

Reply:
Sure